src/profiler.h

```diff
@@ -254,6 +254,10 @@ class Profiler {
         instance()->onGarbageCollectionFinish();
     }
 
+    const StackWalkFeatures* features() const {
```

Comment:
`StackWalkFeatures` is a register-width type, it can be passed by value, not need for a pointer.